### ISSUE TITLE
Add San Bernardino COUNTY California 

### ISSUE THREAD

**Issue by alexjade29** [defund12/defund12.org#1356]
Hi, thanks for your interest in adding your city to the website. Emails on defund12.org are all written by community members. To add your city, you **must** provide:

- Email addresses (at least one) of the government officials you’d like to receive the message
- Detailed message concerning the budgeting decisions at stake in your community

**_Until you provide this content, we cannot post your city._**

Messages should follow our [email style guidelines](https://github.com/defund12/defund12.org/blob/gh-pages/EMAIL_TEMPLATE_STYLE_GUIDE.md). Remember that messages must be specific to your locale and focused on defunding the police. If your message is copied from an external source, please provide a link to the original source.

Your request will be **closed** until you reply with the necessary content. Thank you!

To: 
SupervisorLovingood@sbcounty.gov  assemblymember.obernolte@assembly.ca.gov SupervisorRutherford@sbcounty.gov Supervisor.Rowe@bos.sbcounty.gov supervisor.hagman@bos.sbcounty.gov supervisorgonzales@sbcounty.gov dwert@sbcounty.gov COB@sbcounty.gov da@sbcda.org
Subject: [*** INSERT UNIQUE SUBJECT LINE ***]
Message (Don't forget to replace the [x]'s with your information!)
Hello,
My name is  [YOUR NAME]. I am a resident of  [CITY] and I am writing to you today to demand that San Bernardino County join others across the U.S. in defunding the police.

The Inland Empire and San Bernardino County spend an egregious amount of money on policing and it's time that we reallocate that money back into our communities. 

For decades, we've bore witness to police terrorizing our neighborhoods, rather than lending a hand. We've also witnessed our cities take years to assist at-risk residents, fix roads, and more. We urge you to listen to your constituents and prioritize our basic needs like housing, healthcare, education, economic assistance, and emergency relief. It is your duty to represent us.

Thank you for your time,
[YOUR NAME]

link to original source:
https://twitter.com/suziiiiim/status/1270807297334210560?s=20
![defundsanberco](https://user-images.githubusercontent.com/66755957/84334152-86738700-ab46-11ea-9b6e-98212e48fb4b.PNG)

**Comment by mahrer** [comment]
Hi @alexjade29! Thanks for your interest in adding San Bernardino County to the site. Can you edit your email comment to provide more detail about the city budget to make a more compelling argument, showing the disparities in numbers? To get your submission published quicker, please provide source(s) for the data as well.

Please see our [email style guide](https://github.com/defund12/defund12.org/blob/gh-pages/EMAIL_TEMPLATE_STYLE_GUIDE.md) for some suggestions.

Thank you!

**Comment by silviabotros** [comment]
Here is some info on the budget at the county level(. (source: https://cms.sbcounty.gov/Portals/59/Content/2019-2020/2019-20-Adopted-Budget.pdf)
In the 2019/2020 budget: 1,091.5 Million dollars to "Law and justice"
compared to 63 million for community development and housing


I am not the owner of this PR so I cannot edit the letter directly but i AM an SB county tax payer and would like to help with this

**Comment by mahrer** [comment]
I edited this email with the data cited by @silviabotros:

---

To:
SupervisorLovingood@sbcounty.gov assemblymember.obernolte@assembly.ca.gov SupervisorRutherford@sbcounty.gov Supervisor.Rowe@bos.sbcounty.gov supervisor.hagman@bos.sbcounty.gov supervisorgonzales@sbcounty.gov dwert@sbcounty.gov COB@sbcounty.gov da@sbcda.org

Message (Don't forget to replace the [x]'s with your information!):

Dear San Bernardino County elected officials,

My name is [YOUR NAME] and I am a resident of [CITY]. I am writing to you today to demand that San Bernardino County join others across the U.S. in defunding the police.

The Inland Empire and San Bernardino County spend an egregious amount of money on policing, and it's time that we reallocate that money back into our communities. According to the 2019-2020 Adopted Budget, Law and Justice was allocated $1,068,878,347. In contrast, the Community Development and Housing Agency (which covers homeless services, affordable housing assistance, community development, and infrastructure improvements) was given $63,159,085, and Economic Development (which includes workforce development, resources for job seekers, vocational training, supportive services, and small business assistance) got only $30,513,332.

For decades, we've bore witness to police terrorizing our neighborhoods, rather than lending a hand. We've also witnessed our cities take years to assist at-risk residents, fix roads, and address the critical needs of community members. We urge you to listen to your constituents and prioritize our basic needs like housing, healthcare, education, economic assistance, and emergency relief. It is your duty to represent us.

Thank you for your time,
[YOUR NAME]
[YOUR ADDRESS]
[YOUR EMAIL]
[YOUR PHONE NUMBER]
